File: spring-boot/converter/src/main/java/com/example/converter/controller/MemberController.java
package com.example.converter.controller;

import com.example.converter.entity.Member;
import com.example.converter.repository.MemberRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("/member")
@RequiredArgsConstructor
public class MemberController {

    private final MemberRepository memberRepository;

    @GetMapping("/init")
    public ResponseEntity<String> init() {
        memberRepository.save(Member.builder()
                .name("문성하")
                .regNo("993399-1234567")
                .deleted(true)
                .build());
        memberRepository.save(Member.builder()
                .name("문성하")
                .regNo("993399-1234567")
                .deleted(true)
                .build());
        memberRepository.save(Member.builder()
                .name("문성하")
                .regNo("993399-1234567")
                .deleted(false)
                .build());
        return ResponseEntity.ok().body("ok");
    }
}

File: spring-boot-kotlin/converter/src/main/kotlin/com/example/converter/member/controller/MemberController.kt
package com.example.converter.member.controller

import com.example.converter.member.entity.Member
import com.example.converter.member.repository.MemberRepository
import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/member")
class MemberController(
    private val memberRepository: MemberRepository
) {

    init {
        memberRepository.save(
            Member(
                id = 1,
                name = "문성하",
                regNo = "993399-1234567",
                deleted = true
            )
        )
        memberRepository.save(
            Member(
                id = 2,
                name = "문성하",
                regNo = "993399-1234567",
                deleted = false
            )
        )
        memberRepository.save(
            Member(
                id = 3,
                name = "문성하",
                regNo = "993399-1234567",
                deleted = false
            )
        )
    }

    @GetMapping
    fun findAll(): ResponseEntity<MutableList<Member>> {
        return ResponseEntity.ok().body(memberRepository.findAll())
    }
}